extendeddatasquare.go

```diff
@@ -120,3 +128,34 @@ func (eds *ExtendedDataSquare) deepCopy(codec Codec) (ExtendedDataSquare, error)
 	eds, err := ImportExtendedDataSquare(eds.flattened(), codec, eds.createTreeFn)
 	return *eds, err
 }
+
+// Col returns a column slice.
+// This slice is a copy of the internal column slice.
+func (eds *ExtendedDataSquare) Col(y uint) [][]byte {
+	s := make([][]byte, eds.width)
+	copy(s, eds.colSlice(0, y, eds.width))
+	return s
```

Comment:
nit: append might be more readable/idiomatic? But again, mostly a style question and the `copy` is very explicit.